cli/src/pcluster/config/cluster_config.py

```diff
@@ -1905,6 +1920,15 @@ def instance_type_list(self):
         """Return the list of instance types associated to the Queue."""
         return [compute_resource.instance_type for compute_resource in self.compute_resources]
 
+    @property
+    def instance_types_with_instance_storage(self):
+        """Return a set of instance types in the queue that have instance store."""
+        result = set()
+        for compute_resource in self.compute_resources:
+            if compute_resource.instance_type_info.instance_storage_supported():
```

Comment:
Does this introduce the risk of having a lot of calls to EC2 to retrieve such information?
In case it does, we should cache the info about instance storage support.

Reply:
There is cache for instance_type_info. So no duplicate EC2 is required

Reply:
Great, thanks